homeassistant/components/unifi_access/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The UniFi Access integration."""
+
+from __future__ import annotations
+
+import uiaccessclient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_TOKEN, CONF_HOST, CONF_VERIFY_SSL, Platform
+from homeassistant.core import HomeAssistant
+
+from .coordinator import UniFiAccessDoorCoordinator
+from .data import UniFiAccessData
+
+PLATFORMS: list[Platform] = [Platform.LOCK]
+
+type UniFiAccessConfigEntry = ConfigEntry[UniFiAccessData]  # noqa: F821
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: UniFiAccessConfigEntry) -> bool:
+    """Configure UniFi Access integration."""
+    host = f"https://{entry.data.get(CONF_HOST)}/api/v1/developer"
+    configuration = uiaccessclient.Configuration(
+        host, access_token=entry.data.get(CONF_API_TOKEN)
+    )
+    configuration.verify_ssl = entry.data.get(CONF_VERIFY_SSL)
```

Comment:
why isnt this a parameter?

Reply:
Will fix with wrapper. See comment above.

Reply:
This is now moved to library.